### PR TITLE
Kf 2269 feat jupyter tensorflow cuda full rock

### PR DESCRIPTION
**Pull request by i-chvets**
Tensorflow CUDA full initial ROCK.
    
Summary of changes:
- Initial ROCK version with latest updates and create according to best practices spec.
- Commented out tests as TO-DOs for next step of ROCK tests that will serve as framework for all other tests in Notebook servers.

### CODE REVIEW

File: jupyter-tensorflow-cuda-full/rockcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+      # Install the jupyter-tensorflow CUDA requirements
+      cp ${CRAFT_PART_SRC}/components/example-notebook-servers/jupyter-tensorflow/cuda-requirements.txt cuda-requirements.txt
+      "${CONDA_BIN}/pip" install --no-cache-dir -r cuda-requirements.txt
+
+      # Install the jupyter-tensorflow-full requirements
+      cp ${CRAFT_PART_SRC}/components/example-notebook-servers/jupyter-tensorflow-full/requirements.txt requirements.txt
+      "${CONDA_BIN}/pip" install --no-cache-dir -r requirements.txt
+
+      # Write some config files for activating conda in user sessions
+      echo ". /opt/conda/etc/profile.d/conda.sh" >> "${HOME}/.bashrc"
```
Comment (SamirPS) [inline jupyter-tensorflow-cuda-full/rockcraft.yaml:221]: why a space ?

File: jupyter-tensorflow-cuda-full/rockcraft.yaml [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+      echo ". /opt/conda/etc/profile.d/conda.sh" >> "${CRAFT_PART_INSTALL}/etc/profile"
```
Comment (SamirPS) [inline jupyter-tensorflow-cuda-full/rockcraft.yaml:222]: why a space ?

**Review by SamirPS — CHANGES_REQUESTED** [review]

File: jupyter-tensorflow-cuda-full/rockcraft.yaml (hunk 1 above)
Comment (i-chvets) [reply jupyter-tensorflow-cuda-full/rockcraft.yaml:221]: That 'source' command. That's from original Dockerfile.

File: jupyter-tensorflow-cuda-full/rockcraft.yaml [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+      echo "conda activate base" >> "${HOME}/.bashrc"
+      echo "conda activate base" >> "${CRAFT_PART_INSTALL}/etc/profile"
+
+      # Do some cleanup
+      rm -rf "${HOME}/.wget-hsts" "${HOME}/.cache"
```
Comment (SamirPS) [inline jupyter-tensorflow-cuda-full/rockcraft.yaml:227]: it's hosts no ?

**Review by SamirPS — CHANGES_REQUESTED** [review]

File: jupyter-tensorflow-cuda-full/rockcraft.yaml (hunk 2 above)
Comment (i-chvets) [reply jupyter-tensorflow-cuda-full/rockcraft.yaml:222]: That 'source' command. That's from original Dockerfile.

File: jupyter-tensorflow-cuda-full/rockcraft.yaml (hunk 3 above)
Comment (i-chvets) [reply jupyter-tensorflow-cuda-full/rockcraft.yaml:227]: Yes, it is in build. Again this came verbatim from the Dockerfile.

**Review by SamirPS — APPROVED** [review]

**Review by ca-scribner — APPROVED** [review]

**Pull request merged**